ci-ubuntu-1604.yml

```diff
@@ -0,0 +1,211 @@
+---
```

Comment:
Nit: what about naming this file `travis-ci-ubuntu-1604.yml` to go with the name of the sub directory in the cookbooks i.e. `cookbooks/travis_ci_ubuntu_1604` ?

Reply:
I was mostly following the previous naming conventions, I can look into this renaming.